analyze-wasm/README.md

```diff
@@ -35,6 +35,10 @@ npm install -S @arcjet/analyze-wasm
 
 Use [`@arcjet/analyze`][file-analyze] instead.
 
+## API
+
+Use [`@arcjet/analyze`][file-analyze] instead.
```

Comment:
```suggestion
This is an internal package. Use [`@arcjet/analyze`][file-analyze] instead.
```

Just to be specific in case someone comes across it.